src/sql/common/pathUtilities.ts

```diff
@@ -22,6 +28,9 @@ export function resolveCurrentDirectory(uri: string, rootPath: string): string {
 	} else if (sqlUri.scheme === UNTITLED_SCHEMA) {
 		// if sql file is unsaved/untitled but a workspace is open use workspace root
 		let root = rootPath;
+		if (oldSavePath) {
```

Comment:
should this logic be isolated to the serializer component?  in other words, will the `oldSavePath` variable change the behavior of resolveCurrentDirectory in other use cases then export to CSV, JSON, XLSX leading to inconsistent behavior?